### DISCUSSION TITLE
Can I use github for an organisation?

### DISCUSSION THREAD

**Discussion by Runway27** [community/community#145325] [Other Feature Feedback, Questions, & Ideas]
### Can I use github for an organisation?

I’m running a mechanical engineering company and I am wondering if I can use github for maintaining my companies engineering drawings. 

Since I am having a lot of sensitive data, my requirement is for the repository is to be private to the organisation and be available to only specific systems in the company would it be possible. 

Thanks in advance

**Comment by Awwze** [comment]
Hello Runway27!

Yes, you can absolutely use GitHub for your organization, including maintaining engineering drawings and other sensitive files. GitHub offers several features tailored for businesses and organizations with strict privacy and security requirements:
Private Repositories – Data can be restricted to specific users you grant access to.
GitHub Enterprise – Provides additional security and control features, including IP whitelisting, access controls, and compliance options.
Git LFS (Large File Storage) – Ideal for storing large engineering files like CAD drawings.

I think, I could help you!
